customize.dist/translations/messages.de.js

```diff
@@ -0,0 +1,230 @@
+  define(function () {
+      var out = {};
+
+      // translations must set this key for their language to be available in
+      // the language dropdowns that are shown throughout Cryptpad's interface
+      out._languageName = 'German';
+
+      out.main_title = "Cryptpad: Echtzeitzusammenarbeit, ohne Vorwissen";
+      out.main_slogan = "Einigkeit macht stark - Zusammenarbeit ist der Schlüssel"; // Der Slogan sollte evtl. besser englisch bleiben.
+
+      out.type = {};
+      out.type.pad = 'Pad';
+      out.type.code = 'Code';
+      out.type.poll = 'Umfrage';
+      out.type.slide = 'Präsentation';
+
+      out.errorBox_errorType_disconnected = 'Verbindung verloren';
+      out.errorBox_errorExplanation_disconnected = [
+          'Verbindung zum Server verloren. Zum Wiederverbinden Seite neu laden oder außerhalb dieser Box ',
+          'klicken, um deine Arbeit einzusehen.'
+      ].join('');
+
+      out.common_connectionLost = 'Serververbindung verloren';
+
+      out.disconnected = 'Getrennt';
+      out.synchronizing = 'Synchronisiert';
+      out.reconnecting = 'Verbindung wird neu aufgebaut...';
+      out.lag = 'Lag';
+      out.readonly = 'Nur-Lesen';
+      out.anonymous = "Anonymous";
+      out.yourself = "Du";
+      out.anonymousUsers = "anonyme Nutzer*innen";
+      out.anonymousUser = "anonyme Nutzer*in";
+      out.shareView = "Nur-Lesen URL";
+      out.shareEdit = "Mitarbeits-URL";
+      out.users = "Nutzer*innen";
```

Comment:
I'm a little curious about the asterisk in the middle of these words. Is this normal punctuation?